commit e23a9014fd4d502a419255a83e2479ab804c6f16
Author: Bob Copeland <me@bobcopeland.com>
Date:   Mon Mar 16 22:34:03 2009 -0400

    ath5k: remove dummy PCI "retry timeout" fix
    
    Remove the PCI retry timeout code, for all the same reasons that
    Luis Rodriguez removed it for ath9k.
    
    Changes-licensed-under: 3-Clause-BSD
    
    Cc: Luis Rodriguez <lrodriguez@atheros.com>
    Signed-off-by: Bob Copeland <me@bobcopeland.com>
    Signed-off-by: John W. Linville <linville@tuxdriver.com>

diff --git a/drivers/net/wireless/ath5k/base.c b/drivers/net/wireless/ath5k/base.c
index 8eec155..ef9825f 100644
--- a/drivers/net/wireless/ath5k/base.c
+++ b/drivers/net/wireless/ath5k/base.c
@@ -685,13 +685,6 @@ ath5k_pci_resume(struct pci_dev *pdev)
 	if (err)
 		return err;
 
-	/*
-	 * Suspend/Resume resets the PCI configuration space, so we have to
-	 * re-disable the RETRY_TIMEOUT register (0x41) to keep
-	 * PCI Tx retries from interfering with C3 CPU state
-	 */
-	pci_write_config_byte(pdev, 0x41, 0);
-
 	err = request_irq(pdev->irq, ath5k_intr, IRQF_SHARED, "ath", sc);
 	if (err) {
 		ATH5K_ERR(sc, "request_irq failed\n");
